tests/unit-tests/tests-formatting.php

```diff
@@ -210,4 +210,21 @@ public function give_let_to_num_provider() {
 			array( '1K', 1024, '1K should be equal to 1024' ),
 		);
 	}
+
+	/**
+	 * Test give_validate_nonce function
+	 *
+	 * @since  1.8
+	 *
+	 * @cover  give_validate_nonce
+	 */
+	function test_give_validate_nonce() {
+		$input_nonce = wp_create_nonce( 'give_gateway' );
+
+		/*
+		 * If nonce does not validate successfully then WPDieException throw.
+		 */
+		$this->expectException( 'WPDieException' );
```

Comment:
@ravinderk I'm seeing some failing unit tests for this, can you check into: https://travis-ci.org/WordImpress/Give/builds/182321052